### DISCUSSION TITLE
Camunda BPMN Service task Issue

### DISCUSSION THREAD

**Discussion by Elmira-rahimi-e** [camunda-community-hub/community#116] [Q&A]
Hi community. 
Can anyone help me?
I designed a very simple process with service and script task in Camunda. 
when i start the process and the token reach the service task this message pupped up:

An error happened while submitting the task form :
Cannot submit task form 93e09b31-3b5a-11ee-bda6-160a6459c529: Unable to compile script: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
Script8.groovy: 1: unexpected token:  @ line 1, column 48.
   groovy.json.JsonSlurperClassic
                                 ^

**Comment by xomiamoore** [comment]
Hi @Elmira-rahimi-e, I saw you also posted in the [forums here](https://forum.camunda.io/t/api-service-task-and-gatewaye/46915). That's the best way to get an answer as this GitHub community is for our community projects, plugins, and other tools.
